### ISSUE TITLE
V2: Latest Build File Doesn't Exist "survey-creator-react/survey-creator-react.min.css";

### ISSUE THREAD

**Issue by jpgilchrist** [surveyjs/survey-creator#2710]
### Are you requesting a feature, reporting a bug or ask a question?

bug

### What is the current behavior?

import "survey-creator-react/survey-creator-react.min.css"; throws an error as the file doesn't exist. 

### What is the expected behavior?

it should import and use the correct css

### How would you reproduce the current behavior (if this is a bug)?

package.json:

"survey-creator-react": "^1.9.15-beta.0",

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
import * as Survey from 'survey-react';
import 'survey-core/defaultV2.min.css';
import 'survey-creator-react/survey-creator-react.min.css';

```

**Comment by jpgilchrist** [comment]
![image](https://user-images.githubusercontent.com/5065660/155219289-b8d5c0d2-0395-4f88-bb98-6ac474b6d1e3.png)

**Comment by jpgilchrist** [comment]
I also tried with 1.9.14-beta.1 and some other versions. It was working and suddenly stopped. I rolled back to a specific version of 1.9.3-beta.1 and it does work there.

**Comment by andrewtelnov** [comment]
@jpgilchrist Yes, we move it to "survey-creator-core". We made it as a separate package several releases ago.
Please use the following code:
`import "survey-creator-core/survey-creator-core.min.css";`

Thank you,
Andrew

**Comment by jpgilchrist** [comment]
@andrewtelnov I assume someone will be updating the README then?

**Comment by RomanTsukanov** [comment]
Fixed by #2716
